patches/ninja/0001-Add-support-for-NINJAJOBS.patch

```diff
@@ -0,0 +1,29 @@
+From 7dbf497370dac35a8e2dba6b64eb13989f1ab4fb Mon Sep 17 00:00:00 2001
+From: Dennis Bonke <admin@dennisbonke.com>
+Date: Fri, 29 Apr 2022 20:12:56 +0200
+Subject: [PATCH] Add support for NINJAJOBS
+
+Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
+---
+ src/ninja.cc | 5 +++++
+ 1 file changed, 5 insertions(+)
+
+diff --git a/src/ninja.cc b/src/ninja.cc
+index 471a023..5c1cb3f 100644
+--- a/src/ninja.cc
++++ b/src/ninja.cc
+@@ -227,6 +227,11 @@ void Usage(const BuildConfig& config) {
+ 
+ /// Choose a default value for the -j (parallelism) flag.
+ int GuessParallelism() {
++  int   j = 0;
++  char* jobs = getenv( "NINJAJOBS" );
++  if ( jobs != NULL ) j = atoi( jobs );
++  if ( j > 0 ) return j;
```

Comment:
Why do we patch ninja instead of just using the `-j` flag? Patching ninja doesn't sound really appealing to me.

Reply:
If ninja is invoked by a different build system or a script (for example, qtwebengine, which builds with cmake but then builds chromium with ninja), an user can't pass the `-j` flag. With this patch, we can also use the `NINJAJOBS` environment variable. The patch was sourced from BLFS.

Reply:
When using the `build` subcommand of cmake, you can pass arguments to make/ninja like so: `cmake build -- -jN`

Reply:
Interesting, I didn't know that. In that case it's possible to drop the patch, tho I don't mind keeping it either.